execution/src/windows/execution.cc

```diff
@@ -74,6 +74,11 @@ namespace leatherman { namespace execution {
 
     string which(string const& file, vector<string> const& directories, bool expand)
     {
+        // On Windows is not supported to turn off command expansion by setting expand to false 
```

Comment:
cpplint fails on this line: Line ends in whitespace.  Consider deleting these extra spaces